tools/ci/tests/test_run_tc.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+
+from six import iteritems
+
+from tools.ci import run_tc
+
+
+@pytest.mark.parametrize("msg,expected", [
+    ("Some initial line\n\ntc-jobs:foo,bar", set(["foo", "bar"])),
+    ("Some initial line\n\ntc-jobs:foo, bar", set(["foo", "bar"])),
+    ("tc-jobs:foo, bar   \nbaz", set(["foo", "bar"])),
```

Comment:
How about a test for multiple occurrences of `tc-jobs:`? (Currently, the script ignores those)